dataset/src/main/scala/frameless/TypedDataset.scala

```diff
@@ -438,6 +455,49 @@ class TypedDataset[T] protected[frameless](val dataset: Dataset[T])(implicit val
     selectMany(ca, cb)
   }
 
+  def selectNamed[A: TypedEncoder, B: TypedEncoder, KA <: Symbol, KB <: Symbol](
```

Comment:
Hm, why do we need these? Probably we need them, but right now I am kind of missing the why..

Reply:
By these you mean the `selectNamed` method generally or why does it exist in addition to `selectNamedGeneralized`?

Reply:
I think it was in general. Let me go over this again.